app/views/facility_orders/_merge_order_form.html.haml

```diff
@@ -18,9 +18,8 @@
         class: "datepicker__data string optional"
 
       = label_tag :note, OrderDetail.human_attribute_name(:note)
-      = text_field_tag :note,
+      = text_area_tag :note,
```

Comment:
I don't think `text_area_tag` automatically adds a `maxlength` the way that simple_form does. We should probably add that here.